pretix_wallet/signals.py

```diff
@@ -1,9 +1,21 @@
 from django.dispatch import receiver
-from pretix.base.signals import register_payment_providers
 
-from pretix_wallet.payment import WalletPaymentProvider
+from pretix.base.payment import PaymentException
+from pretix.base.signals import register_payment_providers, order_paid
+from pretix_wallet.models import CustomerWallet
+
+from pretix_wallet.payment import WalletPaymentProvider, _wallet_transaction
 
 
 @receiver(register_payment_providers, dispatch_uid="payment_wallet")
 def register_payment_provider(sender, **kwargs):
     return [WalletPaymentProvider]
+
+
+@receiver(order_paid, dispatch_uid="payment_wallet_order_paid")
+def wallet_order_paid(sender, order, **kwargs):
+    CustomerWallet.create_if_non_existent(sender.organizer, order.customer)
```

Comment:
all of this should only happen if there is a topup product

Reply:
Done